standard/sections/clause_7_normative_text.adoc

```diff
@@ -363,7 +367,22 @@ However, a data producer can use the ``properties.cache`` value to request Globa
 
 include::../recommendations/core/PER_cache.adoc[]
 
-==== 1.15 Properties / Integrity
+==== 1.15 Properties / Cache duration
+
+By default, cached data is stored for a period of 24 hours.   A data producer can use the ``properties.cache-duration`` value to specify that data should be cached for a period other than 24 hours.
```

Comment:
```suggestion
By default, cached data is stored for a period of 24 hours.   A data producer can use the ``properties.cache-duration`` value to request that data  be cached for a period other than 24 hours.
```

Reply:
@tomkralidis Is it true that the GC can decide if they want to cache 24 hours, by the cache-duration value or an other duration?

Reply:
@amilan17 from the Guide:

> A Global Cache shall retain a copy of the core data for a duration compatible with the real-time or near-real-time schedule of the data and not less than 24 hours.

The GC can then decide to cache > 24 hours if needed per `properties.cache-duration`.  For core data, if `properties.cache` is less than 24 hours, the GC would still cache for 24 hours per the above.

Having said this, I think "interaction" specifics would be provided in the Guide (as opposed to WMN proper).